### DISCUSSION TITLE
History auto-purge Script

### DISCUSSION THREAD

**Discussion by weekender23** [nzbgetcom/nzbget#224] [Q&A] [answered]
HI,

I am currently using this script https://forum.nzbget.net/viewtopic.php?f=8&t=2342&sid=4c2f06c622fcf4d4e09d22d1f77c0ef8
to remove certain categories in history. I was wondering if this could be implemented into the newest nzbget program?

thanks.

**Comment by dnzbk** [comment, marked as answer]
Yes, we have already considered this extension and found it useful but haven't gotten to it yet.
Getting more upvotes would increase the priority for development.
